src/main/webapp/style.css

```diff
@@ -2,6 +2,8 @@
   margin-left: auto;
```

Comment:
Do you need this style.css file / is this here by accident? Seems that you can delete it.

Reply:
I left it since index.html uses it, but now I've moved index.html, style.css, and script.js to the home-page folder. Should each page have its own javascript file or just have one for all of them?

Reply:
Each page should have its own JavaScript file(s). Thanks for creating the home-page folder! Great work!